ruma-identity-service-api/src/keys/valid_public_key/v2.rs

```diff
@@ -0,0 +1,56 @@
+//! [GET /_matrix/identity/v2/pubkey/isvalid](https://matrix.org/docs/spec/identity_service/latest#get-matrix-identity-v2-pubkey-isvalid)
```

Comment:
Could you update this to a use the 0.3.0 version of the docs rather than latest (in case we drift from the latest version of the spec). I realize I did this when creating the issue, so that's my bad.

```suggestion
//! [GET /_matrix/identity/v2/pubkey/isvalid](https://matrix.org/docs/spec/identity_service/0.3.0#get-matrix-identity-v2-pubkey-isvalid)
```